src/tiktokbot.ts

```diff
@@ -0,0 +1,283 @@
+// Import modules.
+import { config } from 'dotenv'
+import { Client, Message, Intents, Options, MessageAttachment, MessageEmbed } from 'discord.js'
+import { getVideoMeta as fetch } from 'tiktok-scraper'
+
+// Log <ENV> errors.
+if (config()?.error) throw new Error('Failed to load .env file')
+
+// Default embed color.
+const color: number | string = 0xE18499
+
+// Define global interfaces.
+declare global {
+    var Client: object
+
+    interface String {
+
+        /**
+         * Compare two strings case insensitive.
+         * 
+         * @param [query] The string to compare with "this".
+         * @return boolean
+         * 
+         * @example
+         * "a".compare("b"): false (boolean)
+         * "a".compare("A"): true (boolean)
+         */
+
+        is(query: string): boolean
+    }
+
+    interface MessageExtender extends Message {
+        args: string[]
+        command: string
+    }
+}
+
+/**
+ * Create and login to Discord client with custom properties.
+ * 
+ * @param [] None required.
+ * @example new TikTok()
+ */
+export default class TikTok extends Client {
+    public constructor() {
+        super({
+            intents: ['GUILDS', 'GUILD_MESSAGES'].map((i: string) => (Intents as any).FLAGS[i]),
```

Comment:
```suggestion
            intents: ['GUILDS', 'GUILD_MESSAGES'],
```
It doesn't need to have the other part.

Reply:
[Done](https://github.com/txj-xyz/TikTokBot/pull/2/commits/fd657ba5c856dba4d4693013badf8c13d7ad9c26)